### ISSUE TITLE
NSE not reporting UTD analytics to posthog?

### ISSUE THREAD

**Issue by BillCarsonFr** [element-hq/element-x-ios#2889]
It looks like the NSE target is [not depending on posthog](https://github.com/element-hq/element-x-ios/blob/develop/NSE/SupportingFiles/target.yml)?

Does it means that if a decryption error occurs while doing a NSE sync it won't be reported to posthog?
If true, what would be effort to add reporting for UTD in NSE?

**Comment by stefanceriu** [comment]
> Does it means that if a decryption error occurs while doing a NSE sync it won't be reported to posthog?

Of course, there's nothing automatically happening, nobody's pushing that event through from the NSE

> If true, what would be effort to add reporting for UTD in NSE?

Couple of days if everything goes well
